packages/core/src/index.ts

```diff
@@ -200,7 +201,7 @@ export { DiagnosticsMode, EmitDiagnosticsContext, emitDiagnostics } from './repo
 export { visitMetaCSSDependenciesBFS } from './visit-meta-css-dependencies';
 export { murmurhash3_32_gc } from './murmurhash';
 export { TimedCacheOptions, timedCache } from './timed-cache';
-
+export { atPropertyValidationWarnings } from './validate-at-property';
```

Comment:
don't export internals - import from '@stylable/core/dist/...' when using in tests